tensorrt_llm/serve/openai_disagg_service.py

```
@@ -361,27 +369,71 @@ async def _send_disagg_request_gen_first(
             )
-        )
-        responses = await asyncio.gather(*tasks)
-        return responses[-1]
+
+            queue: asyncio.Queue = asyncio.Queue()
+
+            async def _consume_gen():
+                try:
+                    async for chunk in gen_response:
+                        await queue.put(chunk)
+                except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)